content/bahasa-pemrograman-tidak-penting.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Bahasa pemrograman itu tidak penting!
+desc: Terlalu banyak bahasa pemrograman hanya membuat kita pusing.
+author: Reinaldy Rafli
+github: aldy505
+twitter:
+telegram: aldy505
+date: 2021-12-13
+cover: https://www.codecademy.com/resources/blog/content/images/size/w1600/2020/08/what-programming-language-should-i-learn.png
+categories:
+  - language
+---
+
+Menjelang akhir tahun 2021 ini, sebagai seorang developer kamu pasti tahu bahwa ada banyak sekali programming langage, dari yang secara _syntax_ mudah untuk dipahami seperti Python dan Scratch, dari yang memiliki banyak sekali _parenthesis_ seperti Common Lisp, Clojure, dan Janet, hingga yang sangat cepat sekali secara performa seperti Rust dan C++. Ada lebih banyak programming language lainnya yang dapat kamu [cari tahu sendiri di Wikipedia](https://en.wikipedia.org/wiki/List_of_programming_languages_by_type).
+
+Mungkin tahun depan akan lahir programming language lain, karena seseorang tidak suka dengan cara mengimplementasikan sesuatu pada programming language yang dia gunakan sebelumnya.
+
+Tapi, apa tujuannya? Untuk apa programming language banyak-banyak jika dengan Javascript saja kita bisa membuat frontend, backend, mobile app, hingga [database](https://pouchdb.com/). Begitu juga dengan C# yang bisa untuk membuat frontend, backend, mobile app, terminal app, desktop app, hingga [operating system](https://en.wikipedia.org/wiki/Midori_%28operating_system%29). Untuk apa??
+
+Memilih dan mempelajari programming language baru hanya membuang-buang waktumu saja, apalagi setiap ada language baru, kamu tertarik ke suatu konsep yang membuat language itu sangat menarik dan mempelajarinya secara dalam. Biasanya sih biar keliatan lebih ganteng dibanding orang lain. Tapi memangnya bener ya bisa bikin ganteng? Apalagi kalo fundamentalnya masih nggak kuat.
```

Comment:
Just a quote from a legend:

"A language that doesn't affect the way you think about programming, is not worth knowing"
-- Alan J. Perlis